modules/vmfcore/java/vmfRawBuffer.java

```diff
@@ -0,0 +1,59 @@
+//package com.intel.vmf;
+
+import java.nio.ByteBuffer;
+
+public class vmfRawBuffer
+{
+	static
+    {
+        System.loadLibrary("vmf");
+    }
+	
+    protected ByteBuffer byteBuf;
+    protected int size;
```

Comment:
`ByteBuffer` already has `capacity`, `limit` and `position`;
what `size` duplicates? and why?